packages/frontend/src/components/SavedDashboards/CreateSavedDashboardModal.tsx

```diff
@@ -49,6 +52,20 @@ const CreateSavedDashboardModal: FC<CreateSavedDashboardModalProps> = ({
                         />
                     </FormGroup>
                 </div>
+                <div className={Classes.DIALOG_BODY}>
```

Comment:
I think you can remove this line and the above line and have FormGroups in the same `<div className={Classes.DIALOG_BODY}>` wrapper